test/javascript/components/track/iteration-summary/TestsStatusSummary.test.ts

```diff
@@ -0,0 +1,40 @@
+import React from 'react'
+import { render } from '@testing-library/react'
+import '@testing-library/jest-dom/extend-expect'
+import { TestsStatusSummary } from '../../../../../app/javascript/components/track/iteration-summary/TestsStatusSummary'
+
+test('shows Queued status when testsStatus is queued', async () => {
+  const { getByText } = render(<TestsStatusSummary testsStatus="queued" />)
+
+  expect(getByText('Queued')).toBeInTheDocument()
+})
+
+test('shows Passed status when testsStatus is passed', async () => {
```

Comment:
All these should probably be grabbed by role, and then the _text_ should be tested for.